umm-spec-lib/resources/json-schemas/collection/umm/v1.16.5/umm-search-results-json-schema.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Next time when you update the schemas, please first copy the previous version to the new version and then add and commit then new directory before you make any changes.  This allows reviewers to be able to easily and quickly see the changes you have made to the schemas.

Reply:
can't this still be done either with a fresh branch or by rewriting history in git?